tests/e2e/nodes_scaleup_scaledown.go

```diff
@@ -186,7 +194,11 @@ var _ = ginkgo.Describe("[csi-file-vanilla] [csi-block-vanilla-serialized] Nodes
 		framework.Logf("Allow pods to reschedule on different nodes")
 		// Waiting for pods status to be Ready
 		fss.WaitForStatusReadyReplicas(client, statefulset, replicas)
-		gomega.Expect(fss.CheckMount(client, statefulset, mountPath)).NotTo(gomega.HaveOccurred())
+		if windowsEnv {
```

Comment:
If there is no implementation, Then remove this. Lets skip this test for Windows